.github/workflows/verify.yml

```diff
@@ -1,34 +1,47 @@
 name: Verify Build and Generated Data
 
-# Run on any pull request commit or a push to a main branch.
-# If run on the primary branch, the gradle caches will be updated for all PRs to use.
-on: 
+on:
   pull_request:
+    branches:
```

Comment:
Verify doesn't need to run on branches; nightly handles build verification

Reply:
branches in this case means that it will only run this if the target branch is those 2

Reply:
ah yep that makes sense; in which case could we change the pattern to dev/* and lts/*? thanks!

Reply:
Sorry - I see you've added `dev/*`, but can we also have `lts/*` as that's the pattern we're going to use for LTS builds :) Thanks!